### ISSUE TITLE
Assert if Attribute exists

### ISSUE THREAD

**Issue by jordan-simonovski** [kubeshop/tracetest#2406]
**Describe the enhancement you'd like to see**
I'm looking to use tracetest as a way to analyse our traces and determine trace data quality. As part of doing this, I think it would be super useful to have the ability to not just assert the value of a trace attribute, but also to be able to verify that certain attributes exist.
For example: Writing an assertion to verify that `parentId` is set for a span attribute. 

**Additional context**
Me and another colleague came to the booth at KubeCon and have been interested in trialling this for use at Atlassian

**Comment by schoren** [comment]
Hi @jordan-simonovski. I'm glad to see you're in using Tracetest! That seems like a very useful usecase for the tool, and I think we can totally implement that.

A thing to note about the assertions is that if an attribute doesn't exists, it's  flagged as failed, regardless of the comparsion being made. For example:

![Screenshot 2023-04-19 at 10 42 09](https://user-images.githubusercontent.com/314548/233094177-bfa871eb-c190-4afc-933e-c3ac94f26c01.png)

If you don't care about the values, just that the attribute exists, you can assert that the values IS NOT something that you know it shouldn't be. In this example, I took a custom attribute and asserted that it DOES NOT EQUAL to `""` (literally two quotes). The spans I'm asserting don't contain that attribute, so the test fails. 

In the second assertion, I check that the http status code DOES NOT EQUAL to `""` (literally two quotes) again. In this case the attribute exists, and it has an actual value of `200`. Since `200` does not equal `""`, the assertion passes.

Does this example make sense?

I hope you can use this as a temporary measure until we provide a more clean implementation for this usecase.

**Comment by jordan-simonovski** [comment]
Ah nice! Yeah that's perfectly fine while we're just evaluating it for our own usage.  Thanks for getting back to me so quickly 🙌

**Comment by schoren** [comment]
Absolutely. Let us know if you need any other assistance. You can also reach us in our [discord server](https://discord.gg/6zupCZFQbe)
